src/promptflow/promptflow/_core/openai_injector.py

```diff
@@ -130,14 +140,17 @@ def inject_openai_api():
     # The openai package reads api configs from environment variables only at import time.
     # So we need to update the openai api configs from environment variables here.
     # Please refer to this issue: https://github.com/openai/openai-python/issues/557.
-    openai.api_key = os.environ.get("OPENAI_API_KEY", openai.api_key)
-    openai.organization = os.environ.get("OPENAI_ORGANIZATION", openai.organization)
     openai.api_type = os.environ.get("OPENAI_API_TYPE", openai.api_type)
     openai.api_version = os.environ.get("OPENAI_API_VERSION", openai.api_version)
 
     if IS_LEGACY_OPENAI:
-        openai.api_key_path = os.environ.get("OPENAI_API_KEY_PATH", openai.api_key_path)
         openai.api_base = os.environ.get("OPENAI_API_BASE", openai.api_base)
+        openai.api_key = os.environ.get("OPENAI_API_KEY", openai.api_key)
+        openai.api_key_path = os.environ.get("OPENAI_API_KEY_PATH", openai.api_key_path)
+        openai.organization = os.environ.get("OPENAI_ORGANIZATION", openai.organization)
+    else:
+        openai.azure_endpoint = os.environ.get("AZURE_OPENAI_ENDPOINT", openai.azure_endpoint)
```

Comment:
Add comments on these environment variables?

Reply:
Fixed. The [issue](https://github.com/openai/openai-python/issues/557) has be fixed, so we don't need to inject environment variables with openai versions that greater than 1.0.0.